feat: add page descriptions and update titleType (#340)
carbon-design-system/carbon-website: src/pages/components/inline-loading/code.mdx

---
title: Inline loading

> description: Inline loading spinners are used when performing actions. They help notify user's that their action is being processed.

Reviewer: ```suggestion
description: Inline loading spinners notify a user that their action is being processed.
```
